theme/static/style/style.css

```diff
@@ -452,6 +471,22 @@ pre code {
         padding-left: .8rem;
         text-align: left;
     }
+
```

Comment:
Remove everything in this selector block. Styles for tablet should work fine for mobile phones too.